Core/clim-basic/stream-output.lisp

```diff
@@ -290,32 +290,25 @@
 
 (defun seos-write-newline (stream)
   (let ((medium       (sheet-medium stream))
-        (view-height  (bounding-rectangle-height stream)))
+        ;; Stream region may be +everywhere+.
+        (view-height  (ignore-errors (bounding-rectangle-height stream))))
```

Comment:
Couldn't this be tested with `(region-equal (sheet-region stream) +everywhere+)` instead of `ignore-errors`?

Reply:
it could, I guess it was me being lazy (or overly general) here. Fix pushed.